apps/registry-docs/public/install.sh

```diff
@@ -50,38 +53,62 @@ trap cleanup EXIT INT TERM
 
 # ===== Utilities =====
 
+# Set registry URL based on resource type
+set_registry_url() {
```

Comment:
In the past, there were these two urls with /registry.json & /install.sh each. Now that there's only one url (registry.514.ai), the install script is being merged, but this also impacts the registry.json. I curl'd https://registry.514.ai and it seems to have been updated to include both connectors & pipelines, so we should be able to just use one url here.